src/app.html

```diff
@@ -8,6 +8,7 @@
     <link rel="stylesheet" href="/build/protocol-extra.css" />
     <link rel="stylesheet" href="/tokens.css" />
     <link rel="stylesheet" href="/rally.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css" integrity="sha512-1ycn6IcaQQ40/MKBW2W4Rhis/DbILU74C1vSrLJxCq57o941Ym01SwNsOMqvEBFlcgUa6xLiPY/NS5R+E6ztJQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
```

Comment:
Should we host this ourselves? I think it's safe to do since it's checking subresource integrity, I guess it won't work if you happen to be developing offline.

Reply:
Hmm, true. I'll get the imgs myself when I work on the styles so I hear you.